ipam/space/space_test.go

```diff
@@ -52,12 +52,12 @@ func TestLowlevel(t *testing.T) {
 
 	s := New()
 	wt.AssertEquals(t, s.NumFreeAddresses(), address.Offset(0))
-	ok, got := s.Allocate()
+	ok, got := s.Allocate(0, 1000)
```

Comment:
I think `Space.NumFreeAddresses` is now only used in tests and hence should be moved into this file.